test/org/sagebionetworks/bridge/services/StudyServiceMockTest.java

```diff
@@ -484,12 +492,15 @@ public void createStudyAndUser() throws SynapseException {
 
         // verify
         verify(participantService, times(2)).createParticipant(any(), any(), any(), anyBoolean());
-        verify(participantService).createParticipant(eq(study), eq(mockUser1.getRoles()), eq(mockUser1), eq(true));
-        verify(participantService).createParticipant(eq(study), eq(mockUser2.getRoles()), eq(mockUser2), eq(true));
+        verify(participantService).createParticipant(eq(study), eq(mockUser1.getRoles()), eq(mockUser1), eq(false));
+        verify(participantService).createParticipant(eq(study), eq(mockUser2.getRoles()), eq(mockUser2), eq(false));
         verify(participantService, times(2)).requestResetPassword(eq(study), eq(mockIdentifierHolder.getIdentifier()));
         verify(mockSynapseClient, times(2)).newAccountEmailValidation(any(), eq(SYNAPSE_REGISTER_END_POINT));
         verify(service).createStudy(study);
         verify(service).createSynapseProjectTeam(TEST_ADMIN_IDS, study);
+        
+        assertTrue(projectCaptor.getValue().getName().contains(TEST_PROJECT_NAME));
+        assertTrue(teamCaptor.getValue().getName().contains(TEST_TEAM_NAME));
```

Comment:
Minor: Is it worth using startsWith() as a stronger test than contains()?

Minor: Is it worth checking that the name isn't exactly TEST_NNNN_NAME to make sure we're appending that extra random bit?

Reply:
Switched to startsWith(). The TEST_PROJECT_NAME/TEST_TEAM_NAME constants include the strings we're appending to the study name as converted.